### PR TITLE
refactor!(kubernetes/workloads): change replica ready query to percent

### PR DESCRIPTION
**Pull request by francovp**
This PR change the replica ready monitor query. 

Now the query is modified to obtain the percentage of available replicas and this can be used across any size of Replicaset

### CODE REVIEW

File: caas/kubernetes/workload/monitors-k8s-workload.tf [hunk 1]
```diff
@@ -95,7 +95,7 @@ resource "datadog_monitor" "replica_ready" {
     ${var.replica_ready_time_aggregator}(${var.replica_ready_timeframe}):
       max:kubernetes_state.replicaset.replicas_desired${module.filter-tags.query_alert} by {${local.replica_group_by}} -
       max:kubernetes_state.replicaset.replicas_ready${module.filter-tags.query_alert} by {${local.replica_group_by}}
-    + 1 < ${var.replica_ready_threshold_critical}
+      > ${var.replica_ready_threshold_critical}
```
Comment (coneking) [inline caas/kubernetes/workload/monitors-k8s-workload.tf:98]: Maybe the alert could be something like this

`( (replicas_desired - replicas_ready) / replicas_desired ) * 100`


And de trigger alert 

`>= 50 Critical`
`> 25 Warning`

Comment (francovp) [reply caas/kubernetes/workload/monitors-k8s-workload.tf:98]: Added in [7e99778](https://github.com/fiftech/terraform-datadog-monitors/pull/6/commits/7e99778e53358fc1f7541fc1bfc2ad89ebd46cc6)

**Comment by jcapote22** [comment]
LGTM

**Comment by coneking** [comment]
LGTM

**Review by coneking — APPROVED** [review]

**Review by vigohe — APPROVED** [review]

**Pull request merged**